database/transaction.go

```diff
@@ -103,6 +103,39 @@ func (tx Transaction) GetTable(name string) (*Table, error) {
 	}, nil
 }
 
+// RenameTable renames a table named old to new.
```

Comment:
Renaming is always from something to something else.
```suggestion
// RenameTable renames a table.
```